src/app/features/courses-page/courses-page.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { SharedModule } from 'src/app/shared';
+
+import { CoursesPageComponent } from './courses-page.component';
+import { CourseSearchComponent, CoursesListComponent, CourseComponent, AddCourseButtonComponent } from './components';
+
+
+@NgModule({
+  declarations: [
+    CoursesPageComponent,
+    CourseSearchComponent,
+    CoursesListComponent,
+    CourseComponent,
+    AddCourseButtonComponent
+  ],
+  imports: [
+    SharedModule
```

Comment:
delete this SharedModule. Create a separate module for each entity instead

Reply:
@SteveRusin SharedModule was deleted.